### PR TITLE
Add externalId to Person and show it on PersonEditForm on German Systems

### PR DESCRIPTION
**Pull request by fuatse**
Add Person.externalId in domain model and show in PersonEditForm on German Systems #2670

### CODE REVIEW

**Comment by MateStrysewske** [comment]
Hey! Please rebase your branch to the current development branch to get rid of the merge conflicts before we review your pull request. Thanks!

**Comment by MateStrysewske** [comment]
@fuatse Please also add the field to the mobile app. You don't necessarily have to include it in the forms, but please add it to the entity and database (as explained in the "how to add a new field" guide).

**Comment by MateStrysewske** [comment]
@fuatse Hey! Sorry that it took me so long to react here. Can you get rid of the conflicts once more? I'll review the PR asap afterwards to make sure that you don't have to do it again.

**Comment by fuatse** [comment]
> @fuatse Hey! Sorry that it took me so long to react here. Can you get rid of the conflicts once more? I'll review the PR asap afterwards to make sure that you don't have to do it again.

Hello @MateStrysewske No worries, I completely understand how hard it can be managing such situations. Just did the update for issue #2670 . Hope the build is successful

**Comment by MateStrysewske** [comment]
Two things:

- [ ] Seems like you accidentally removed the external ID from the person form in the web app again
- [ ] In the Android app, you also need to increase the DatabaseHelper.DATABASE_VERSION whenever you add upgrade code. The DATABASE_VERSION always has to be one number higher than the highest CASE statement in the upgrade part.

**Comment by fuatse** [comment]
> Two things:
> 
> * [ ]  Seems like you accidentally removed the external ID from the person form in the web app again
> * [ ]  In the Android app, you also need to increase the DatabaseHelper.DATABASE_VERSION whenever you add upgrade code. The DATABASE_VERSION always has to be one number higher than the highest CASE statement in the upgrade part.

@MateStrysewske I took out the field after I misunderstood your request for the mobile app. I just added it again. I have equally updated the database version for the mobile app for issue #2670 , Thanks for the heads up

File: sormas-app/app/src/main/java/de/symeda/sormas/app/backend/person/Person.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * along with this program. If not, see <https://www.gnu.org/licenses/>.
  */
 
-package de.symeda.sormas.app.backend.person;
+package de.symeda.sormas.app.backend.person;cd
```
Comment (MateStrysewske) [inline sormas-app/app/src/main/java/de/symeda/sormas/app/backend/person/Person.java:16]: ;)

**Review by MateStrysewske — CHANGES_REQUESTED** [review]
One more rebase and a fix for the little problem I've encountered and I'll merge this!

File: sormas-app/app/src/main/java/de/symeda/sormas/app/backend/person/Person.java (hunk 1 above)
Comment (fuatse) [reply sormas-app/app/src/main/java/de/symeda/sormas/app/backend/person/Person.java]: Grrrrr.... Changes pushed @MateStrysewske

**Pull request merged**